src/zope/applicationcontrol/tests/test_runtimeinfo.py

```diff
@@ -119,14 +142,56 @@ def test_Uptime(self):
 
         # get the uptime the current implementation calculates
         test_uptime = runtime_info.getUptime()
+        self.assertAlmostEqual(asserted_uptime, test_uptime, delta=time_tolerance)
 
-        self.assertTrue(abs(asserted_uptime - test_uptime) < time_tolerance)
+    @unittest.skipIf(not sys.path, "Need entries on path")
```

Comment:
We're providing false assurance in many cases, since we're not enabling branch coverage tracking.  I don't think that's a problem.

(Probably not worth changing now.  I just wanted to understand your motivation.)